help.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: help-redirect
+permalink: /help/
+---
+
+<script>
+  window.location.replace("/user-guide");
+</script>
```

Comment:
Hmm you might want to consider using a fallback mechanism if javascript is not enabled.
A simple link, or even a meta refresh tag will do the trick already.

Reply:
Good idea, by any chance do you know how to do a meta refresh in markdown?

Reply:
Well you'll have to use normal html, but this'll require you to create a new template or something.
IIRC there was a way to use layouts with parameters, so there should be no need to have a second almost identical template

Reply:
Hmm, at that rate IMO a link and relying on JS is probably sufficient. It's worth us keeping the overhead low. IIRC, virtually all browsers have JS enabled now, we'd be doing the template work just for the bots, and even then, they've the link they can follow.

Reply:
Thoughts? @RoiEXLab

Reply:
I'm not talking about browser support here, I'm mostly concerned about user using addons like noscript etc. that mostly use a default-disabled setting for individual scripts, so you have to opt-in to use them.
So some sort of indicator "please visit page xyz, redirect didn't work for some reason" would be nice so that people don't just have to look at an empty page if anything goes wrong

Reply:
Agreed, a link like "Page moved to 'user-guides'" would be easy enough.
Small delay here, my jekyll broke, had to spend some time fixing it.

Reply:
@RoiEXLab now updated: 535c241

Again, great suggestion/consideration.

With the link, if the redirect does not do its thing, the page will look like this:
![Screenshot from 2019-11-27 18-12-26](https://user-images.githubusercontent.com/12397753/69771441-a52da580-1141-11ea-89a1-05d7764d3aa2.png)